client/rest-high-level/build.gradle

```diff
@@ -83,6 +83,7 @@ processTestResources {
   from({ zipTree(configurations.restSpec.singleFile) }) {
     include 'rest-api-spec/api/**'
   }
+  from(project(':client:rest-high-level').file('src/test/resources'))
```

Comment:
I have added a certificate chain ( `testClient.crt`, `testIntermediateCA.crt` and `testRootCA.crt` files) as resources to these two projects.